### DISCUSSION TITLE
URL scheme deal with space ？

### DISCUSSION THREAD

**Discussion by lengyijun** [xiaoyifang/goldendict-ng#1885] [General]
It looks like url scheme doesn't deal with space correctly.

I configure `<a href="goldendict://{{Front}}">{{Front}}</a>` in anki.

It works nice for single word, but failed at `write away to`

**Comment by lengyijun** [comment]
%20 doesn't work also

**Comment by lengyijun** [comment]
Qt: Session management error: Authentication Rejected, reason : MIT-MAGIC-COOKIE-1 authentication rejected
Goldendict-ng 24.09.0 at 2024-10-21T00:11:06Z
Qt 6.7.2 GCC 14.2.1 20240910 manjaro linux 6.6.54-2-MANJARO x86_64-little_endian-lp64
Flags: MAKE_ZIM_SUPPORT MAKE_CHINESE_CONVERSION_SUPPORT NO_TTS_SUPPORT no_ffmpeg_player

**Comment by shenlebantongying** [comment]
I still don't know what's wrong, but my previous investigation showed that Anki potentially has bug. The `%20` works in other places.

https://forum.freemdict.com/t/topic/31642/8

**Comment by xiaoyifang** [comment]
https://github.com/xiaoyifang/goldendict-ng/blob/8a92cbf34ad90a9dc7e45461d792b2c3c03098c6/src/main.cc#L320

startsWith->contains?

I guess 
goldendict://host/path,   maybe there are some restriction on the host part.

**Reply by xiaoyifang** [reply]
or maybe try to configure like this   `<a href="goldendict:///{{Front}}">{{Front}}</a>`  . notice the `///` not `//`

**Reply by lengyijun** [reply]
Worked for me, thx
